PKGBUILD

```diff
@@ -1,42 +1,48 @@
 # Maintainer: Grégoire Seux <grego_aur@familleseux.net>
 # Contributor: Dean Galvin <deangalvin3@gmail.com>
+# Contributor: NicoHood <archlinux {cat} nicohood {dog} de>
+
 pkgname="home-assistant"
-pkgdesc='Home Assistant is an open-source home automation platform running on Python 3'
+pkgdesc='Open-source home automation platform running on Python 3'
 pkgver=0.53.1
-pkgrel=1
+pkgrel=3
 url="https://home-assistant.io/"
-license=('MIT')
+license=('APACHE')
 arch=('any')
 replaces=('python-home-assistant')
 makedepends=('python-setuptools')
 # NB: this package will install additional python packages in /var/lib/hass/lib depending on components present in the configuration files.
-depends=('python>=3.4' 'python-pip' 'python-requests>=2.14.2' 'python-yaml' 'python-pytz>=2017.2' 'python-vincenty' 'python-jinja>=2' 'python-voluptuous>=0.9.3' 'python-netifaces' 'python-webcolors' 'python-async-timeout>=1.3.0' 'python-aiohttp>=2.2.5' 'python-jinja>=2.9.5' 'python-yarl>=0.11.0' 'python-chardet>=3.0.4' 'python-astral')
+depends=('python' 'python-pip' 'python-requests>=2.14.2' 'python-yaml' 'python-pytz>=2017.2'
+         'python-vincenty' 'python-jinja>=2' 'python-voluptuous>=0.9.3' 'python-netifaces'
+         'python-webcolors' 'python-async-timeout>=1.3.0' 'python-aiohttp>=2.2.5'
+         'python-jinja>=2.9.5' 'python-yarl>=0.11.0' 'python-chardet>=3.0.4' 'python-astral')
```

Comment:
OPT: since you edit this line, could you remove the duplicate python-jinja? thanks

Reply:
Will do.